src/client/sandbox/node/document.js

```diff
@@ -16,16 +16,20 @@ export default class DocumentSandbox extends SandboxBase {
         this.nodeSandbox = nodeSandbox;
     }
 
-    _isUninitializedIframeWithoutSrc (window) {
+    _isUninitializedIframeWithoutSrc () {
         try {
-            return window !== window.top && isIframeWithoutSrc(window.frameElement) &&
-                   !IframeSandbox.isIframeInitialized(window.frameElement);
+            return this.window !== this.window.top && isIframeWithoutSrc(this.window.frameElement) &&
+                   !IframeSandbox.isIframeInitialized(this.window.frameElement);
         }
         catch (e) {
             return false;
         }
     }
 
+    _isDocumentBeCleanedAfterWrite () {
+        return this.document.readyState !== 'loading';
```

Comment:
this state indicates that document will be cleaned after ```document.write```, because page alredy loaded